frontend/beCompliant/src/components/tableActions/TableFilter.tsx

```diff
@@ -25,6 +27,14 @@ export const TableFilter = ({
     activeFilterValue
   );
 
+  if (
+    filterName === 'Svar' &&
+    type !==
+      (OptionalFieldType.OPTION_SINGLE || OptionalFieldType.OPTION_MULTIPLE)
```

Comment:
Mener denne gjør noe annet enn det du vil at den skal gjøre.

Her evalurerer den `(OptionalFieldType.OPTION_SINGLE || OptionalFieldType.OPTION_MULTIPLE)` først som alltid evaluerer til `OptionalFieldType.OPTION_SINGLE` siden den er "truthy".

Deretter sjekker du om type er forskjellig fra den.

Burde heller gjøre
```suggestion
    (type !== OptionalFieldType.OPTION_SINGLE ||
    type !== OptionalFieldType.OPTION_MULTIPLE)
```